nova/compute/api.py

```diff
@@ -321,22 +321,27 @@ def _check_num_instances_quota(self, context, instance_type, min_count,
         vram_mb = 0
         req_ram = max_count * (instance_type['memory_mb'] + vram_mb)
 
+        quota_key_instances = 'instances'
+        if instance_type['extra_specs'].get('quota:separate', 'false') == 'true':
+            quota_key_instances = 'instances_' + instance_type['flavorid']
+        deltas = { quota_key_instances: max_count }
+        reserve_cpu_ram = instance_type['extra_specs'].get('quota:no_reserve_cpu_ram', 'false') != 'true'
```

Comment:
Hmm, not sure about `no_reserve_cpu_ram`.
My suggestions: `no_reserve_resources`, `skip_resources`, `ignore_resources`, `instance_count_only`
Not that I think, that they are terribly better...

Amongst them, I currently like, if only slightly, `instance_count_only`, as it is a positive description of what it does, instead of what it not does. Maybe `account_instance_count_only`? But that gets a bit long...

Maybe we brainstorm a bit more about the name for that.

Reply:
I changed the name to `quota:instance_only`.